requirements.txt

```diff
@@ -1,3 +1,4 @@
 distribute==0.6.45
 lxml==3.5.0
 python-Levenshtein==0.10.2
+l20n==4.0.0a1
```

Comment:
This is not the right place for the requirement. Check `requirements/` directory.